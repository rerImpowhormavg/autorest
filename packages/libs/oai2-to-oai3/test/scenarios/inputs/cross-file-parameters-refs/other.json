{
  "parameters": {
    "QueryParm": {
      "in": "query",
      "name": "MyQueryParam",
      "required": true,
      "schema": {
        "$ref": "#/definitions/MyQuerySchema"
      }
    },
    "HeaderParam": {
      "in": "query",
      "name": "MyHeaderParam",
      "required": true,
      "schema": {
        "$ref": "#/definitions/MyHeaderSchema"
      }
    }
  },
  "definitions": {
    "MyQuerySchema": {
      "type": "string"
    },
    "MyHeaderSchema": {
      "type": "string"
    }
  }
}
