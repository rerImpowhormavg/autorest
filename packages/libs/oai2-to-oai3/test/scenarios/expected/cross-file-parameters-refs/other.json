{
  "servers": [],
  "components": {
    "parameters": {
      "QueryParm": {
        "name": "MyQueryParam",
        "in": "query",
        "required": true,
        "schema": {
          "items": {
            "$ref": "#/components/schemas/MyQuerySchema"
          }
        }
      },
      "HeaderParam": {
        "name": "MyHeaderParam",
        "in": "query",
        "required": true,
        "schema": {
          "items": {
            "$ref": "#/components/schemas/MyHeaderSchema"
          }
        }
      }
    },
    "schemas": {
      "MyQuerySchema": {
        "type": "string"
      },
      "MyHeaderSchema": {
        "type": "string"
      }
    }
  }
}